paddlecloud/paddlejob/views.py

```diff
@@ -121,3 +120,63 @@ def delete(self, request, format=None):
         except ApiException, e:
             logging.error("error deleting pserver pods: %s" % str(e))
         return utils.simple_response(200, delete_status)
+
+
+class LogsView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, format=None):
+        """
+        Get logs for jobs
+        """
+        username = request.user.username
+        namespace = notebook.utils.email_escape(username)
+
+        jobname = request.query_params.get("jobname")
+        num_lines = request.query_params.get("n")
+        worker = request.query_params.get("w")
+        job_pod_list = client.CoreV1Api().list_namespaced_pod(namespace, label_selector="job-name=%s"%jobname)
```

Comment:
trainer job label: https://github.com/PaddlePaddle/cloud/blob/develop/paddlecloud/paddlejob/paddle_job.py#L96  is `paddle-job: {jobname}`, chagne `label_selector="job-name=%s"%jobname ` to `label_selector="paddle-job=%s"%jobname ` ?